docs-site/scripts/push-algolia-index.js

```
@@ -0,0 +1,160 @@
+    `Done! Pushed ${records.length} records to Algolia index "${INDEX_NAME}"`,
+  );
+}
+
+async function waitForTask(baseUrl, headers, taskID) {
+  const appId = headers["X-Algolia-Application-Id"];
+  const url = `https://${appId}.algolia.net/1/indexes/${INDEX_NAME}/task/${taskID}`;
+  for (let i = 0; i < 30; i++) {
+    const resp = await fetch(url, { headers });
+    if (resp.ok) {
+      const data = await resp.json();
+      if (data.status === "published") return;
```

ESLint

[error] Expected { after 'if' condition.

(curly)